pontoon/teams/static/js/info.js

```diff
@@ -0,0 +1,48 @@
+/* global $ */
+$(function() {
+  var container = $('#main .container');
+
+  container.on('click', '.edit-info', function(e) {
+    e.preventDefault();
+    var infoBox = container.find('.info');
+    var content = infoBox.html();
+    var textArea = $('.read-write-info textarea').val($.trim(content));
+    $('.edit-info').hide();
```

Comment:
We should change all `$('.className')` selectors to `container.find('.className')`. This script is active on all team page tabs and it's too dangerous to assume that we will never add an element with one of these class names to any of the tabs.